data/scenarios/README.md

```diff
@@ -271,6 +271,26 @@ table.
 | `system`      | `False`  | `boolean`             | Whether the robot is a "system" robot.  System robots can do anything, without regard for devices and capabilities. System robots are invisible by default.                                                                                                                                                                                                                                                                                                            |
 | `heavy`       | `False`  | `boolean`             | Whether the robot is heavy.  Heavy robots require `tank treads` to `move` (rather than just `treads` for other robots).                                                                                                                                                                                                                                                                                                        |
 
+
+#### Base robot
+
+There must be at most one **base** robot in the world. Since concrete robots can be created
+either via the `loc` attribute or via the map and palette, a cascade of precedence is used to
+determine which robot shall be the base:
+
+1. Prefer those robots defined with a `loc` in the Scenario file
+    1. If multiple robots define a `loc`, use the robot that is defined first within the
+       Scenario file.
+    2. Note that if a robot is both given a `loc` AND is specified in the world map, then
+       two instances of the robot shall be created.  The instance with the `loc` shall be
+       preferred as the base.
+2. Fall back to robots generated from templates via the map and palette.
+    1. If multiple robots are specified in the map, prefer the one that is defined first
+       within the Scenario file.
+    2. If multiple robots are instantiated from the *same* template, then prefer
+       the one closest to the upper-left of the screen,
+       with higher rows given precedence over columns.
```

Comment:
Please make this shorter.

Only a few people will read this long description about which robot will be chosen as the base.
It will also be annoying to keep up-to-date.

I would suggest not explaining the behaviour in such detail but instead giving an example that will make the robot base for sure. From what I understand, there are two strategies:
1. when the robot is first with a location in the list, it is base for sure
2. when no robots have a location, base is the leftmost topmost on the map

When there are more robots in the topmost leftmost cell, base is implementation-defined.

Reply:
Ok, I've shortened the user-facing text, and relegated the verbose text to a comment in the code.